.circleci/config.yml

```
@@ -317,10 +317,9 @@ jobs:
     steps:
       - checkout
       - run:
-          name: setup version from git tag
+          name: Setup version from git tag
```

Minor nitpick, but to use "set up" as a verb we need to write it as separate words:

```suggestion
          name: Set up version from git tag
```